test/compiler/inference.jl

```diff
@@ -1529,8 +1529,7 @@ end
 
 # approximate static parameters due to unions
 let T1 = Array{Float64}, T2 = Array{_1,2} where _1
-    inference_test_copy(a::T) where {T<:Array} = ccall(:jl_array_copy, Ref{T}, (Any,), a)
```

Comment:
And in any case, this change is breaking the test (the test may pass but it doesn't test what it was supposed to test anymore). You can replace it with `jl_value_ptr`. (though since it doesn't run the function, a non-existing function or a pointer as callee should work too).

Reply:
I can just revert this on the test, since jl_array_copy isn't actually removed here it's fine.